underworld/systems/_advectiondiffusion.py

```diff
@@ -154,7 +301,7 @@ def _setup(self):
             if isinstance( cond, uw.conditions.NeumannCondition ):
 
                 ### -VE flux because of the FEM discretisation method of the initial equation
-                negativeCond = uw.conditions.NeumannCondition( fn_flux=-1.0*cond.fn_flux,
```

Comment:
We need to be a bit careful about the implications of doing this.  In particular, if we expose this function as a property so the user can reuse it (or replace it), do we exposed the negatived version?  If we do, it's confusing for the user, if we don't then we need to be careful about always applying the negative under the hood when required.  

Should we just neg it at the C level where it's applied?